src/nl/rubensten/texifyidea/inspections/latex/LatexNonBreakingSpaceInspection.kt

```diff
@@ -25,9 +25,17 @@ import nl.rubensten.texifyidea.util.parentOfType
  */
 open class LatexNonBreakingSpaceInspection : TexifyInspectionBase() {
 
+    companion object {
+
+        /**
+         * All commands that should not have a forced breaking space.
+         */
+        val IGNORED_COMMANDS = setOf("\\citet", "\\citet*", "\\Citet", "\\Citet*")
```

Comment:
I guess the second \citet and \Citet should be changed to \citep and \Citep

Reply:
No? Only `\citet` variants must be ignored?

Reply:
Oh I see I misread the original issue, sorry